### DISCUSSION TITLE
GitHub Projects is just painful to use on mobile

### DISCUSSION THREAD

**Discussion by jordie23** [community/community#11057] [Mobile]
It’s such a pain. Spend 1 minute using projects and you can’t get anything useful done. It’s frustrating to use. I don’t know where to start if you want specifics. Mobile support is just not there. Is this being addressed at all? I feel like this should have been a basic thing by now.

**Comment by mxie** [comment]
Hi @jordie23 - we don't currently support Projects in the app but it _is_ on our roadmap. Were you using Projects in the mobile browser by any chance?
